terraso_backend/tests/conftest.py

```diff
@@ -46,6 +46,11 @@ def logged_client(access_token):
     return Client(HTTP_AUTHORIZATION=f"Bearer {access_token}")
 
 
+@pytest.fixture
+def not_logged_client(access_token):
```

Comment:
question: what does `not_logged` mean?

Reply:
@paulschreiber mean that it is a fixture for a client to generate requests simulating a request from a not logged in user

Reply:
Ah. Please call it `not_logged_in_client` then.